dump.go

```diff
@@ -0,0 +1,97 @@
+package jet
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"reflect"
+	"strings"
+)
+
+// dumpAll returns
+//  - everything in Runtime.context
+//  - everything in Runtime.variables
+//  - everything in Runtime.set.globals
+//  - everything in Runtime.blocks
+func dumpAll(a Arguments, depth int) reflect.Value {
+	var b bytes.Buffer
+	var vars VarMap
+
+	ctx := a.runtime.context
+	fmt.Fprintln(&b, "Runtime.context:")
+	fmt.Fprintf(&b, "\t%s %q\n", ctx.Type(), ctx)
```

Comment:
could we use `%#v` as format of the value we print in all the printf calls? that should make the output more clear: https://play.golang.org/p/6OWfcKwTPmJ

Reply:
resolved